### ISSUE TITLE
HttpStatusCode display node only shows name.

### ISSUE THREAD

**Issue by Psychpsyo** [Yellow-Dog-Man/Resonite-Issues#188]
### Is your feature request related to a problem? Please describe.

I'm always frustrated when plugging a ProtoFlux wire of type ``System.Net.HttpStatusCode`` into a display node because it doesn't show the full, proper name of the status code. Usually it leaves out the number and spaces so a 404 for example is simply shown as "NotFound".

The one exception I've found seems to be ``418 I'm a teapot``, which is shown as "418" instead of its name.

### Describe the solution you'd like

The node should display both the name and number for the status code. So something like "404 Not Found" instead of "NotFound".

### Describe alternatives you've considered

I'm sure I could extract the number from the enum(?) somehow but that's way too tedious for something as small as this.

### Additional Context

The number is very nice to have, since it lets you easily tell what kind of response code you got. 1XX is info, 2XX is success, 3XX are redirects, 4XX are your fault, 5XX is their fault.

**Comment by shadowpanther** [comment]
The better way would be to have a node that separates the number and the message of the status code, I think.

**Comment by Psychpsyo** [comment]
> The better way would be to have a node that separates the number and the message of the status code, I think.

Having access to both (probably as an int/uint and a string) would be neat but it doesn't solve the problem of the display node being kinda bad for it. I don't want to get out a separate node just to look at info that should be on the display node.

**Comment by Frooxius** [comment]
I don't think this is something we can do that much about without some weirdness.

The Display node is generic and the status code is an enum. The Display node just shows an enum, however it's named. The enum itself is defined in .NET libraries, so we don't control how they're named. 

However one thing that helps is you can use the node to convert enum to int, which will give you the status code.
